src/Sulu/Bundle/HttpCacheBundle/Resources/varnish/sulu.vcl

```diff
@@ -48,6 +48,10 @@ sub sulu_recv {
     if (req.http.Authorization) {
         return (pass);
     }
+
+    if (req.http.Cookie ~ "_ss") {
```

Comment:
Not necessary, since the documentation includes exactly that file anyway: https://docs.sulu.io/en/2.0/cookbook/caching-with-varnish.html?highlight=varnish#varnish-configuration